docs/admin/admin-schema.yaml

```diff
@@ -23,5 +36,17 @@ definitions:
         type: string
       email:
         type: string
-      isEmailConfirmed:
+      active:
+        type: boolean
+      blocked:
         type: boolean
+      dateOfInvitation:
+        type: string
+        format: date-time
```

Comment:
check the format. should be [ISO 8601]

Reply:
date-time – the date-time notation as defined by RFC 3339, for example, 2017-07-21T17:32:28Z